src/payloads/AppUser.ts

```diff
@@ -46,3 +46,23 @@ export class AppUserRolesResponse {
   @IsEnum(AppUserRole)
   role: string;
 }
+
+export class AppUserProfileResponse {
+  @IsString()
+  firstName: string;
+
+  @IsString()
+  lastName: string;
+
+  @IsString()
+  email: string;
+
+  @IsString()
+  major: string;
+
+  @IsString()
+  graduationYear: string;
+
+  @IsInstance(InductionClass)
+  inductionClass: InductionClass;
```

Comment:
eh we can probably just grab the name of the induction class or the quarter or sth

Reply:
I think most people will remember the quarter in which they inducted. What they won't remember is the name of their induction class.

Reply:
yeah we can grab the quarter that's a good point